.github/workflows/test.yml

```diff
@@ -12,12 +12,6 @@ jobs:
     strategy:
       matrix:
         include:
-          - case-name: qutip@dev.major
```

Comment:
I think this case should stay but become `qutip@master` and then the case below should become `qutip@qutip-4-7-X`.

Reply:
We will still make fixes to 4.7.X and qutip-qip should still test against it.